assets/src/admin/paired-browsing/app.js

```diff
@@ -299,6 +299,9 @@ class PairedBrowsingApp {
 			{ passive: true },
 		);
 
+		// Scrolling is not synchronized if `scroll-behavior` is set to `scroll`.
+		win.document.querySelector( 'html' ).style.setProperty( 'scroll-behavior', 'auto', 'important' );
```

Comment:
Great idea to set this with JS. And TIL `setProperty`!

```suggestion
		win.document.documentElement.style.setProperty( 'scroll-behavior', 'auto', 'important' );
```

Reply:
Please also check what happens if someone sets `scroll-behavior` on the `body` instead.

Reply:
Setting `scroll-behavior`set to `smooth on the `body` element doesn't have the same effect.